Gopkg.toml

```diff
@@ -0,0 +1,11 @@
+[[constraint]]
```

Comment:
Completely optional, but I think this project could be good to start trying out go modules. What do you think? 😄